src/mnode/mnode-governance.h

```diff
@@ -200,4 +200,7 @@ class CMasternodeGovernance
     void Clear();
 
     void UpdatedBlockTip(const CBlockIndex *pindex);
-};
+
+private:
+    bool CanVote(std::string& strErrorRet) const;
```

Comment:
noexcept

Reply:
Please make it protected & virtual - this way it is easier to write tests. Will just mock it later on.